CHANGELOG.md

```diff
@@ -1,122 +1,124 @@
+- We've removed singletons as a class specifier, so you need to remove this if you use it. To replicate the behavior of former singletons, ensure that your Actor is spawned once by a single server-side worker in your deployment.
+- We've renamed `OnConnected` and `OnConnectionFailed` on `SpatialGameInstance` to `OnSpatialConnected` and `OnSpatialConnectionFailed`. They are now also blueprint-assignable.
 
 ### Features:
-- Unreal Engine `4.24.3` is now supported. You can find the `4.24.3` version of our engine fork [here](https://github.com/improbableio/UnrealEngine/tree/4.24-SpatialOSUnrealGDK-preview).
-- Added a new variable `QueuedOutgoingRPCWaitTime`. Outgoing RPCs will now be dropped if: more than `QueuedOutgoingRPCWaitTime` time has passed; the worker is never expected to become authoritative in zoning/offloading scenario; the Actor is being destroyed.
-- In local deployments of the Example Project you can now launch Simulated Players in one click. Running `LaunchSimPlayerClient.bat` will launch a single Simulated Player client. Running `Launch10SimPlayerClients.bat` will launch 10.
-- Added an AuthorityIntent component to be used in the future for UnrealGDK code to control loadbalancing.
-- Added support for the UE4 Network Profile to measure relative size of RPC and Actor replication data.
-- Added a VirtualWorkerTranslation component to be used in future UnrealGDK loadbalancing.
-- Added partial framework for use in future UnrealGDK controlled loadbalancing.
-- Add SpatialToggleMetricsDisplay console command.  bEnableMetricsDisplay must be enabled in order for the display to be available.  You must then must call SpatialToggleMetricsDisplay on each client that wants to view the metrics display.
-- Enabled compression in modular-udp networking stack
-- Switched off default rpc-packing. This can still be re-enabled in SpatialGDKSettings.ini
+- We now support Unreal Engine `4.24.3`. You can find the `4.24.3` version of our engine fork [here](https://github.com/improbableio/UnrealEngine/tree/4.24-SpatialOSUnrealGDK-preview).
+- We've added a new variable: `QueuedOutgoingRPCWaitTime`. Outgoing RPCs are now dropped in the following three scenarios: more than `QueuedOutgoingRPCWaitTime` time has passed, the worker instance is never expected to become authoritative in zoning/offloading scenario, or the Actor is being destroyed.
+- In local deployments of the Example Project, you can now launch simulated players in one click. Running `LaunchSimPlayerClient.bat` launches a single simulated player client. Running `Launch10SimPlayerClients.bat` launches ten simulated player clients.
+- We've added support for the UE4 Network Profiler to measure relative size of RPC and Actor replication data.
+- We've added a `SpatialToggleMetricsDisplay` console command. You must enable `bEnableMetricsDisplay` in order for the display to be available. You must then must call `SpatialToggleMetricsDisplay` on each client that wants to view the metrics display.
```

Comment:
Does "console command" mean a CLI command, or something else (like something inside the Editor)?

Reply:
Also, could you clarify "that wants to view the metrics display"?

Reply:
The client as in the UnrealClient, the instance of the game. The developer/player needs to call the console command on their client.

Reply:
Is it the developer or the player who does this, or could it be either? (The reason I'm asking is that in the release notes we're, talking to the developer - "You need to do X" - so we don't want to suddenly have "you" refer to the player.)

Is the following correct?

> You _[i.e. the developer]_ need to call `SpatialToggleMetricsDisplay` on each client that you want the metrics display to be visible for.

Reply:
You refers to dev